astronomer/providers/google/cloud/sensors/bigquery.py

```diff
@@ -0,0 +1,78 @@
+"""This module contains Google Big Query sensors."""
+from typing import Any, Dict, Optional
+
+from airflow.exceptions import AirflowException
+from airflow.providers.google.cloud.sensors.bigquery import BigQueryTableExistenceSensor
+
+from astronomer.providers.google.cloud.triggers.bigquery import (
+    BigQueryTableExistenceTrigger,
+)
+
+
+class BigQueryTableExistenceSensorAsync(BigQueryTableExistenceSensor):
+    """
+    Checks for the existence of a table in Google Big Query.
+     :param project_id: The Google cloud project in which to look for the table.
+        The connection supplied to the hook must provide
+        access to the specified project.
+     :param dataset_id: The name of the dataset in which to look for the table.
+        storage bucket.
+     :param table_id: The name of the table to check the existence of.
+     :param gcp_conn_id: The connection ID used to connect to Google Cloud.
+     :param bigquery_conn_id: (Deprecated) The connection ID used to connect to Google Cloud.
+        This parameter has been deprecated. You should pass the gcp_conn_id parameter instead.
+     :param delegate_to: The account to impersonate using domain-wide delegation of authority,
+        if any. For this to work, the service account making the request must have
+        domain-wide delegation enabled.
+     :param impersonation_chain: Optional service account to impersonate using short-term
+        credentials, or chained list of accounts required to get the access_token
+        of the last account in the list, which will be impersonated in the request.
+        If set as a string, the account must grant the originating account
+        the Service Account Token Creator IAM role.
+        If set as a sequence, the identities from the list must grant
+        Service Account Token Creator IAM role to the directly preceding identity, with first
+        account from the list granting this role to the originating account (templated).
+     :param polling_interval: The interval in seconds to wait between checks table existence.
+    """
+
+    def __init__(
+        self,
+        gcp_conn_id: str = "google_cloud_default",
+        polling_interval: float = 5.0,
+        **kwargs: Any,
+    ) -> None:
+        super().__init__(**kwargs)
+        self.polling_interval = polling_interval
+        self.gcp_conn_id = gcp_conn_id
+
+    def execute(self, context: Dict[str, Any]) -> None:
+        """Airflow runs this method on the worker and defers using the trigger."""
+        self.defer(
+            timeout=self.execution_timeout,
+            trigger=BigQueryTableExistenceTrigger(
+                dataset_id=self.dataset_id,
+                table_id=self.table_id,
+                project_id=self.project_id,
+                poll_interval=self.polling_interval,
+                google_cloud_conn_id=self.gcp_conn_id,
```

Comment:
Any reason why we are translating connection name from `gcp_conn_id` to `google_cloud_conn_id`. Is it possible to keep it consistent either `google_cloud_conn_id` or `gcp_conn_id`?

Reply:
Fixed to `gcp_conn_id`